_data/events/Hyderabad-JUG.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "JUG Hyderabad GDCR 17",
+  "url": "https://www.meetup.com/jughyderabad/events/244680395",
+  "moderators": [
+    "@jbuddha"
```

Comment:
Are you sure [`@jbuddha`](https://twitter.com/jbuddha) is your twitter handle? :)

Reply:
sorry, I thought we have to give github user id, will submit another pull request

Reply:
updated the pull request.please merge. thanks for correction